{
    "osv": {
        "package": {
            "name": "python-pysaml2"
        },
        "published": "2017-01-12",
        "references": [
            {
                "type": "DSA",
                "url": "https://www.debian.org/security/2017/dsa-3759"
            }
        ]
    },
    "uvi": {
        "experimental": {
            "advisory_type": "DSA-3759-1",
            "other_identifiers": {
                "cve": [
                    "CVE-2016-10149"
                ]
            },
            "package_affected": [
                {
                    "distribution_affected": "jessie",
                    "fixed in": "2.0.0-1+deb8u1"
                },
                {
                    "distribution_affected": "stretch) and unstable (sid",
                    "fixed in": "3.0.0-5"
                }
            ],
            "package_name": "python-pysaml2",
            "processed_timestamp": "2021-07-18T03:09:08.151415Z",
            "uvi_script_name": "./www.debian.org-processing.py",
            "uvi_script_version": "0.0.5",
            "vulnerability_status": "yes"
        },
        "product_name": "Linux",
        "vendor_name": "Debian"
    }
}